rct229/rulesets/ashrae9012019/data_fns/table_G3_5_1_fns.py

```diff
@@ -0,0 +1,60 @@
+from rct229.rulesets.ashrae9012019.data import data
+from rct229.rulesets.ashrae9012019.data_fns.table_utils import (
+    find_osstd_table_entry,
+    find_osstd_table_entries,
+)
+from rct229.schema.config import ureg
+
+
+# Make sure this line is a sorted (ascending order) list
+capacity_threshold_list = [0, 65000, 135000, 240000, 760000, 999999999]
+
+
+def table_G3_5_1_lookup(capacity):
```

Comment:
I agree Tables G3.5.1-4 can all use the same schema where `most_conservative_efficiency` may be None. However, Table G3.5.5 I have set up to return a list of dictionaries where each dictionary follows the schema. This is because Warm-air furnace, gas-fired <225,000 Btu/h has 2 options for minimum efficiency: 78% AFUE or 80% Et.

Reply:
hmm, I think in that case, the returned data type for table G3.5.5 can be `List[G351TableSearchInfo]`.
Since `most_conservative_efficiency` is None, you can safely reuse this type for the element inside the list.

Reply:
Thanks for mentioning about this. I will check it in detail.